### DISCUSSION TITLE
Difficulties trying to add a test user with specific access role

### DISCUSSION THREAD

**Discussion by KB-WEB-DEVELOPMENT** [Budibase/budibase#5976] [Help] [answered]
Hi,

I completed a real estate agency app with 11 tables offering CRUD functionalities and created two new access roles : agent, client

1) New access roles : https://ibb.co/c1f7Xx6

2) I would like to offer real estate agencies the possibility to test the app and log in either as an admin, agent or as a client , decided to go with the Basic user boarding option (and later the Email boarding option if possible) :

https://ibb.co/6Bnh85L

3)  I am a little lost as to how to add a new user with his email address, set his password as well as access role.
The error window I am getting is shown here : 

https://ibb.co/cLqcLkh

I would be very grateful if someone could point out to me the steps needed to add a user (email address/password) so that
the application can be tested by an external company.

(I contacted Budibase Sales team with this question because there app itself does not need to be improved anymore for now.)

Thanks for your time and patience.

KBW

**Comment by melohagan** [comment, marked as answer]
Hi @KB-WEB-DEVELOPMENT 

In the error message it appears that the email address isn't correct, it ends in **.comb** instead of **.com**

Try going through the process again and see if the error persists.

**Comment by KB-WEB-DEVELOPMENT** [comment]
@melohagan : thanks a lot , it worked. Truly amazing .
